<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>SetRenderWindowSizeGUI</class>
	<widget class="QWidget" name="vrSetRenderWindowSizeGUI">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>600</width>
				<height>200</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Render Window Size</string>
		</property>
		<property name="group" stdset="0">
			<string>Scripts</string>
		</property>
		<layout class="QGridLayout" name="gridLayout">
		
			<item row="0" column="0">
				<widget class="QLabel" name="sep0">
					<property name="text">
						<string>Current Size - Ratio</string>
					</property>
				</widget>
			</item>
			<item row="0" column="1">
				<widget class="QLabel" name="currentSizeLabel">
					<property name="text">
						<string>XxY - X:Y</string>
					</property>
				</widget>
			</item>
			<item row="1" column="0">
				<widget class="QPushButton" name="_SIZUPD_">
					<property name="text">
						<string>Update</string>
					</property>
					<property name="toolTip">
						<string>Update the current Size and Ratio Label</string>
					</property>
				</widget> 
			</item>
			
			<item row="2" column="0">
				<widget class="QLabel" name="sep0">
					<property name="text">
						<string>Custom Ratio</string>
					</property>
					<property name="toolTip">
						<string>Set the render windows size with custom ratio calculated with the input
Note then if the oirizontal or vertical measure are bigger then the maximum render window size X or Y
the resultant render window will be calculated with the maximun size to fit the render window but with the calculated ratio
You can put either a pixel measure (1024 and 768) or a ratio (16 and 9)</string>
					</property>
				</widget>
			</item>
			<item row="3" column="0">
				<widget class="QLineEdit" name="Orizontal_Input">
					<property name="text">
						<string>0</string>
					</property>
					<property name="toolTip">
						<string>input the Orizontal measure of the final image (pixel or X ratio)</string>
					</property>
				</widget>
			</item>
			<item row="3" column="1">
				<widget class="QLineEdit" name="Vertical_Input">
					<property name="text">
						<string>0</string>
					</property>
					<property name="toolTip">
						<string>input the Vertical measure of the final image (pixel or Y ratio)</string>
					</property>
				</widget>
			</item>
			<item row="4" column="0">
				<widget class="QPushButton" name="_CUSUPD_">
					<property name="text">
						<string>Set Custom Ratio</string>
					</property>
					<property name="toolTip">
						<string>Set the render windows size with custom ratio calculated with the input
Note then if the oirizontal or vertical measure are bigger then the maximum render window size X or Y
the resultant render window will be calculated with the maximun size to fit the render window but with the calculated ratio
You can put either a pixel measure (1024 and 768) or a ratio (16 and 9)</string>
					</property>
				</widget> 
			</item>
			
			
			<item row="5" column="0">
				<widget class="QLabel" name="sep0">
					<property name="text">
						<string>Fixed Sizes</string>
					</property>
				</widget>
			</item>
			<item row="6" column="0">
				<widget class="QPushButton" name="_1080O_">
					<property name="text">
						<string>1920x1080 - 16/9 Orizzontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 16/9 orizontal. 1920x1080</string>
					</property>
				</widget>
			</item>
			<item row="6" column="1">
				<widget class="QPushButton" name="_1080V_">
					<property name="text">
						<string>1920x1080 - 16/9 Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 16/9 vertical. 1080x1920</string>
					</property>
				</widget>
			</item>
			<item row="7" column="0">
				<widget class="QPushButton" name="_720O_">
					<property name="text">
						<string>1280x720 - 16/9 Orizzontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 16/9 orizontal. 1280x720</string>
					</property>
				</widget>
			</item>
			<item row="7" column="1">
				<widget class="QPushButton" name="_720V_">
					<property name="text">
						<string>1280x720 - 16/9 Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 16/9 Vertical. 720x1280</string>
					</property>
				</widget>
			</item>
			
			<item row="10" column="0">
				<widget class="QLabel" name="sep0">
					<property name="text">
						<string>Dynamic sizes</string>
					</property>
				</widget>
			</item>
			<item row="11" column="0">
				<widget class="QPushButton" name="_D169O_">
					<property name="text">
						<string>16/9 Orizontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 16/9 Orizontal</string>
					</property>
				</widget>
			</item>
			<item row="11" column="1">
				<widget class="QPushButton" name="_D169V_">
					<property name="text">
						<string>16/9 Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 16/9 Orizontal</string>
					</property>
				</widget>
			</item>
			<item row="12" column="0">
				<widget class="QPushButton" name="_D43O_">
					<property name="text">
						<string>4/3 Orizontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 4/3 Orizontal</string>
					</property>
				</widget>
			</item>
			<item row="12" column="1">
				<widget class="QPushButton" name="_D43V_">
					<property name="text">
						<string>4/3 Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 4/3 Vertical</string>
					</property>
				</widget>
			</item>
			<item row="13" column="0">
				<widget class="QPushButton" name="_D32O_">
					<property name="text">
						<string>3/2 Orizontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 3/2 Orizontal</string>
					</property>
				</widget>
			</item>
			<item row="13" column="1">
				<widget class="QPushButton" name="_D32V_">
					<property name="text">
						<string>3/2 Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 3/2 Vertical</string>
					</property>
				</widget>
			</item>
			<item row="14" column="0">
				<widget class="QPushButton" name="_D21O_">
					<property name="text">
						<string>2:1 Orizontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 2:1 Orizontal</string>
					</property>
				</widget>
			</item>
			<item row="14" column="1">
				<widget class="QPushButton" name="_D21V_">
					<property name="text">
						<string>2:1 Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 2:1 Vertical</string>
					</property>
				</widget>
			</item>
			<item row="15" column="0">
				<widget class="QPushButton" name="_DAXO_">
					<property name="text">
						<string>Ax Orizontal</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of Ax Orizontal</string>
					</property>
				</widget>
			</item>
			<item row="15" column="1">
				<widget class="QPushButton" name="_DAXV_">
					<property name="text">
						<string>Ax Vertical</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of Ax Vertical</string>
					</property>
				</widget>
			</item>
			<item row="16" column="0">
				<widget class="QPushButton" name="_D11_">
					<property name="text">
						<string>1:1</string>
					</property>
					<property name="toolTip">
						<string>Set the render Windows Size to a ratio of 1:1</string>
					</property>
				</widget>
			</item>
			
			<item row="20" column="0">
				<widget class="QLabel" name="sep0">
					<property name="text">
						<string>Utilities</string>
					</property>
				</widget>
			</item>
			<item row="21" column="0">
				<widget class="QPushButton" name="_DYNA_">
					<property name="text">
						<string>Dynamic Render Window Size</string>
					</property>
					<property name="toolTip">
						<string>Set the render window size to dynamic</string>
					</property>
				</widget>
			</item>
			<item row="22" column="0">
				<widget class="QPushButton" name="_UNDOCKRW_">
					<property name="text">
						<string>Undock Render Window</string>
					</property>
					<property name="toolTip">
						<string>Undock The Main Render Window</string>
					</property>
				</widget>
			</item>
			<item row="22" column="1">
				<widget class="QPushButton" name="_DOCKRW_">
					<property name="text">
						<string>Dock Render Window</string>
					</property>
					<property name="toolTip">
						<string>Dock The Main Render Window</string>
					</property>
				</widget>
			</item>
			<item row="23" column="0">
				<widget class="QPushButton" name="_TOGFULLRW_">
					<property name="text">
						<string>Toggle Fullscreen Render Window</string>
					</property>
					<property name="toolTip">
						<string>Toggle ON or OFF Fullscreen Render Window</string>
					</property>
				</widget>
			</item>
	
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
